cobrawap/pipeline/stage05_channel_wave_characterization/configs/config_template.yaml

```diff
@@ -7,7 +7,7 @@ STAGE_NAME: 'stage05_channel_wave_characterization'
 PROFILE: 'dataset_key'
 
 # Name of the output file
-STAGE_OUTPUT: 'channel-wise_measures.csv'
+STAGE_OUTPUT: 'channel_wise_measures.csv'
```

Comment:
I don't think this should be changed. We would need to evaluate if this is helpful or more consistent in any way.

Reply:
We previously changed the stage name, by using "channel_wave" instead of "channel-wave". So I was acting here coherently.

Reply:
The naming convention for the stage names requires no use of '-' since it can cause issues when chaining together the pipeline from the stages. However, this is separate from conventions for file names. I think with the current naming conventions, this change in file names wouldn't make it more consistent. However, we could think about standardizing the naming of stages and their output files some more, also for the other stages.
For the stage05 this may entail also adapting the stage names. One possibility would be, for example, `stage05_wave_wise_characterization` :-> `wave_wise_characterization.csv` and `stage05_channel_wise_characaterization` :-> `channel_wise_characterization.csv`.
Such changes may, however, influence existing pipeline applications. Therefore, I would postpone such improvements of the naming conventions to version 0.2.